examples/connext_dds/custom_transport/c/FileTransport.c

```diff
@@ -1482,6 +1482,11 @@ static RTI_INT32 NDDS_Transport_FILE_create_sendresource_srEA(
 
     *sendresource_out = NULL;
 
+    /* Check if dest_address_in is null */
+    if (dest_address_in == NULL) {
+        return 0;
```

Comment:
this is something that is tested in the function `NDDS_Transport_Address_is_multicast` so it is not really needed. If this throws a warning and we need to check `!= NULL`, we should add a message error

Reply:
Yes, but the static analyzer threw a warning, so I fixed it.